crates/winch/src/builder.rs

```diff
@@ -1,17 +1,30 @@
 use crate::compiler::Compiler;
 use anyhow::Result;
+use cranelift_codegen::settings;
 use std::sync::Arc;
 use target_lexicon::Triple;
 use wasmtime_environ::{CompilerBuilder, Setting};
 use winch_codegen::isa;
 
+/// Compiler builder.
 struct Builder {
+    /// Target triple.
     triple: Triple,
```

Comment:
`isa::Builder` has a `triple()` method.

Reply:
Yeah, I noticed that, but I don't have a concrete use-case for it at the moment. I'll add one when needed.